cypress/integration/page.spec.js

```diff
@@ -64,509 +64,509 @@
+  //         .find('a')
+  //         .should('not.exist')
+  //     });
+
+  //     it('should have a text name if the father is not in the table', () => {
+  //       page.people().eq(1)
+  //         .find('td').eq(5)
+  //         .find('a')
+  //         .should('not.exist')
+  //     });
+
+  //     it('should have an empty cell if the motherName is not given', () => {
+  //       page.people().eq(20)
+  //         .find('td').eq(4)
+  //         .should('have.text', '-')
+  //     });
+
+  //     it('should have an empty cell if the fatherName is not given', () => {
+  //       page.people().eq(20)
+  //         .find('td').eq(5)
+  //         .should('have.text', '-')
+  //     });
+  //   });
+  // });
+
+  // describe('#/people/:correct-slug page', () => {
+  //   beforeEach(() => {
+  //     page.mockPeople();
+  //     page.visit('/#/people/emma-de-milliano-1876');
+  //   });
+
+  //   it('should have correct address', () => {
+  //     page.assertHash('#/people/emma-de-milliano-1876');
+  //   });
+
+  //   it('should have People nav link active', () => {
+  //     page.nav()
+  //       .contains('a', 'People')
+  //       .and('have.class', ACTIVE_NAV_LINK_CLASS);
+  //   });
+
+  //   it('should have only correct title', () => {
+  //     page.assetTitle('People Page');
+  //   });
+
+  //   it('should have people table', () => {
+  //     page.peopleTable().should('exist');
+  //   });
+
+  //   it('should have one selected person', () => {
+  //     page.people().eq(1)
+  //       .should('have.class', SELECTED_PERSON_CLASS);
+
+  //     cy.get('[data-cy="person"].' + SELECTED_PERSON_CLASS)
+  //       .should('have.length', 1);
+  //   });
+
+  //   it('should highlight person with given slug', () => {
+  //     page.visit('/#/people/jan-van-brussel-1714');
+
+  //     page.people().eq(3)
+  //       .should('have.class', SELECTED_PERSON_CLASS)
+  //   });
+
+  //   it('should allow to select another person', () => {
+  //     page.visit('/#/people/jan-van-brussel-1714');
+
+  //     page.people().eq(3)
+  //       .find('td').eq(0)
+  //       .find('a')
+  //       .click();
+
+  //     page.assertHash('#/people/jan-van-brussel-1714')
+
+  //     page.people().eq(3)
+  //       .should('have.class', SELECTED_PERSON_CLASS);
+
+  //     page.people().eq(1)
+  //       .should('not.have.class', SELECTED_PERSON_CLASS);
+  //   });
+  // });
+
+  // describe('#/people/:wrong-slug page', () => {
+  //   beforeEach(() => {
+  //     page.mockPeople();
+  //     page.visit('/#/people/non-existing-slug');
+  //   });
+
+  //   it('should have correct address', () => {
+  //     page.assertHash('#/people/non-existing-slug');
+  //   });
+
+  //   it('should have People nav link active', () => {
+  //     page.nav()
+  //       .contains('a', 'People')
+  //       .and('have.class', ACTIVE_NAV_LINK_CLASS);
+  //   });
+
+  //   it('should have only correct title', () => {
+  //     page.assetTitle('People Page');
```

Comment:
The function `assetTitle` is likely a typo and should be `assertTitle` to correctly describe its purpose of asserting the page title.